package.json

```diff
@@ -78,6 +78,7 @@
     "@mui/material": "^5.6.2",
     "@mui/styles": "^5.8.0",
     "i18next": "^21.8.11",
+    "plausible-tracker": "^0.3.8",
```

Comment:
Ignore tracking for now and remove this, we'll have to figure out how to do it in the future and add it to all relevant components.